index.html

```diff
@@ -134,6 +137,329 @@ <h1 class="banner-header banner-content">
+              ></object>
+            </li>
+            <li><p class="overline-text-2">Back End Dev</p></li>
+            <li>
+              <object
+                class="overline-ellipse"
+                data="assets/illustrations/ellipse-1.svg"
+              ></object>
+            </li>
+            <li><p class="overline-text-2">2015</p></li>
+          </ul>
+        </div>
+
+        <p class="body-text-2">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+
+        <div class="work-card-techstack">
+          <ul>
+            <li><p class="sub-text-2">html</p></li>
+            <li><p class="sub-text-2">css</p></li>
+            <li><p class="sub-text-2">javaScript</p></li>
+          </ul>
+        </div>
+
+        <div>
+          <button type="button" class="work-card-button button-text-1">See Project</button>
+        </div>
+      </div>
+
+      <!--Card-4-->
+      <div class="works-card">
+        <img
+          class="portfolio-images"
+          src="assets/images/portfolio-image-4.png"
+          alt="portfolio-image-1"
+        />
+        <h2 class="header-text-1">Tonic</h2>
+
+        <div class="work-card-overline">
+          <ul>
+            <li><p class="overline-text-1">CANOPY</p></li>
+            <li>
+              <object
+                class="overline-ellipse"
+                data="assets/illustrations/ellipse-1.svg"
+              ></object>
+            </li>
+            <li><p class="overline-text-2">Back End Dev</p></li>
+            <li>
+              <object
+                class="overline-ellipse"
+                data="assets/illustrations/ellipse-1.svg"
+              ></object>
+            </li>
+            <li><p class="overline-text-2">2015</p></li>
+          </ul>
+        </div>
+
+        <p class="body-text-2">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+
+        <div class="work-card-techstack">
+          <ul>
+            <li><p class="sub-text-2">html</p></li>
+            <li><p class="sub-text-2">css</p></li>
+            <li><p class="sub-text-2">javaScript</p></li>
+          </ul>
+        </div>
+
+        <div>
+          <button type="button" class="work-card-button button-text-1">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <!--ABOUT AREA-->
+    <section id="About" class="body-content-2">
+      <!--About Content-->
+      <div class="about-content">
+        <h1 class="banner-header">About Myself</h1>
+
+        <p class="body-text-2">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+
+        <p class="head-text-1">LET'S CONNECT</p>
```

Comment:
According to the review guidelines, this text should be wrapped inside a `<hx>` tag. Kindly fix this.

<img width="438" alt="Screen Shot 2022-05-26 at 14 42 30" src="https://user-images.githubusercontent.com/82426682/170564991-c83421f9-1c32-4bd0-8813-948d724f3df1.png">